### ISSUE TITLE
Getting error on project with coverlet installed

### ISSUE THREAD

**Issue by danpetitt** [b3b00/coverlet-action#7]
I have a project which runs `dotnet test` locally fine without any issues; but in a Workflow just spits out a failure without any reasons why (I understand the xml cant be found cos dotnet test failed, so that message is bogus):

```
run dotnet test -c Debug -p:coverletOutput=TestResults/coverage.opencover.xml -p:CollectCoverage=true -p:CoverletOutputFormat=opencover -p:Exclude=\"\" ./Tests/Coderanger.ImageInfo.Tests.csproj

Error: dotnet test failure Command failed: dotnet test -c Debug -p:coverletOutput=TestResults/coverage.opencover.xml -p:CollectCoverage=true -p:CoverletOutputFormat=opencover -p:Exclude=\"\" ./Tests/Coderanger.ImageInfo.Tests.csproj
Error: error occured : coverage file not found at ./Tests/TestResults/coverage.opencover.xml
```

My project file has the packages referenced:
```xml
  <ItemGroup>
    <PackageReference Include="coverlet.collector" Version="3.1.1">
      <PrivateAssets>all</PrivateAssets>
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
    </PackageReference>
    <PackageReference Include="coverlet.msbuild" Version="3.1.1">
      <PrivateAssets>all</PrivateAssets>
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
    </PackageReference>
    <PackageReference Include="ReportGenerator" Version="5.0.3">
      <PrivateAssets>all</PrivateAssets>
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
    </PackageReference>
  </ItemGroup>
```

And I have also added the same packages to the test project.

I have another library project which is working fine; so I do not understand what is wrong with this one ... could you give me some ideas why its failing.

**Comment by b3b00** [comment]
hello, could you give me a link to your repo so i can look at it ?

**Comment by danpetitt** [comment]
Its a private repo but I have invited you directly.

**Comment by danpetitt** [comment]
I think its to do with the paths/working directory

**Comment by b3b00** [comment]
When trying to run your tests I get errors on ShouldMatchChildSnapshot. 
I think the same occurs on GH action. 
I've got to look at why coverlet-action is hiding the output when unit tests fail.
And definitely the error message ``` coverage file not found at ./Tests/TestResults/coverage.opencover.xml``` is of no use in this case.

Looking at it

**Comment by danpetitt** [comment]
Thanks; that certainly makes sense, issue is probably to do with some endian differences with my decoding failing the test.

But thanks for looking at the reporting of this.

**Comment by b3b00** [comment]
I've got published a minor version 1.1.8. Could you try it. I 'm not sure it will solve the hidden display, but it worh a try.

**Comment by danpetitt** [comment]
Unfortunately didn’t work:

```
Run b3b00/coverlet-action@1.1.8
with:
    testProject: ./Tests/Coderanger.ImageInfo.Tests.csproj
    outputFormat: opencover
    output: TestResults/coverage.opencover.xml
    threshold: 50
  env:
    DOTNET_ROOT: /home/runner/.dotnet

run dotnet test -c Debug -p:coverletOutput=TestResults/coverage.opencover.xml -p:CollectCoverage=true -p:CoverletOutputFormat=opencover -p:Exclude=\"\" ./Tests/Coderanger.ImageInfo.Tests.csproj

Error: dotnet test failure Command failed: dotnet test -c Debug -p:coverletOutput=TestResults/coverage.opencover.xml -p:CollectCoverage=true -p:CoverletOutputFormat=opencover -p:Exclude=\"\" ./Tests/Coderanger.ImageInfo.Tests.csproj

Error: error occured : coverage file not found at ./Tests/TestResults/coverage.opencover.xml

coverlet test done.
```

**Comment by b3b00** [comment]
@danpetitt , release 1.1.8.2 seems to solve the issue. I 've tested it on a personal private repo that acts as yours.
Tell me if it 's ok

**Comment by danpetitt** [comment]
Cool; I will give it a go, thx

**Comment by b3b00** [comment]
GH actions seems not to know 4 positions versions... I published version 1.1.9

**Comment by danpetitt** [comment]
Wonderful, have some output now on the failing tests.

Just a display issue with the github runner being in USA date format which is why the snapshots were failing.

Thanks for your help and the fix!
